### ISSUE TITLE
Agari does not work with more than 14 tiles in hand?

### ISSUE THREAD

**Issue by Enerccio** [MahjongRepository/mahjong#6]
>>> melds = [TilesConverter.string_to_34_array(sou='1111')]
    >>> tiles =  TilesConverter.string_to_34_array(sou='111122233344455')
    >>> a.is_agari(tiles, melds)
    False
    >>> a.is_agari(tiles)
    False

15 tiles hand should be winnable because of that kan

**Comment by Nihisil** [comment]
Yeah, you can't just add 1111 tiles to the hand. The library needs to know if that kan was declared or not, otherwise, these tiles could be used in different forms.

**Comment by Enerccio** [comment]
yep, I just wish it was explained more. It's quite hard to use this library at the beginning lol
